Error HIGH_ACCURACY of undefined

After installing it on capacitor + vuejs, following the basic steps and updating the MainActivity,
 the android emulator gives this error

`09-21 20:23:48.506 11267-11267/com.edeyapp.delivery E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.edeyapp.delivery, PID: 11267
    java.lang.RuntimeException: Unable to start service com.getcapacitor.community.bglocation.LocationUpdatesService@34128a7 with Intent { act=com.getcapacitor.community.bglocation.goforeground cmp=com.edeyapp.delivery/com.getcapacitor.community.bglocation.LocationUpdatesService VirtualScreenParam=Params{mDisplayId=-1, null, mFlags=0x00000000)} }: java.lang.NullPointerException: println needs a message
        at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:4079)
        at android.app.ActivityThread.access$2400(ActivityThread.java:221)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1897)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:158)
        at android.app.ActivityThread.main(ActivityThread.java:7224)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
     Caused by: java.lang.NullPointerException: println needs a message
        at android.util.Log.println_native(Native Method)
        at android.util.Log.w(Log.java:250)
        at com.getcapacitor.community.bglocation.LocationUpdatesService.getNotification(LocationUpdatesService.java:242)
        at com.getcapacitor.community.bglocation.LocationUpdatesService.onStartCommand(LocationUpdatesService.java:122)
        at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:4062)
        at android.app.ActivityThread.access$2400(ActivityThread.java:221) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1897) 
        at android.os.Handler.dispatchMessage(Handler.java:102) 
        at android.os.Looper.loop(Looper.java:158) 
        at android.app.ActivityThread.main(ActivityThread.java:7224) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120) 
`

Please can you tell me what i am doing wrong.

--- a/android/src/main/java/com/getcapacitor/community/bglocation/LocationUpdatesService.java
+++ b/android/src/main/java/com/getcapacitor/community/bglocation/LocationUpdatesService.java
@@ -239,7 +239,8 @@ public class LocationUpdatesService extends Service {
       intent = new Intent(this,  Class.forName(mainActivityName));
     } catch (Exception ex) {
       intent = new Intent(this,  getApplication().getClass());
-      Log.w(TAG, ex.getLocalizedMessage());
+      Log.w(TAG, "Cannot create main activity Intent: " + ex.getLocalizedMessage());
+      Log.d(TAG, "Stack: " + Log.getStackTraceString(ex));
     }
     intent.setAction(Intent.ACTION_VIEW);
     PendingIntent pendingIntent = PendingIntent.getActivity(this, 0, intent, 0);

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "capacitor-background-geolocation",
-  "version": "0.0.5",
+  "version": "0.0.6",
   "description": "A native plugin to get geolocation updates also when app is in background",
   "main": "dist/plugin.js",
   "module": "dist/esm/index.js",

